parsl/monitoring/monitoring.py

```
@@ -300,24 +300,32 @@ def monitor_wrapper(f,
         Wrap the Parsl app with a function that will call the monitor function and point it at the correct pid when the task begins.
         """
         def wrapped(*args, **kwargs):
-            command_q = Queue(maxsize=10)
+            # Send first message to monitoring router
+            try:
+                monitor(os.getpid(),
+                        task_id,
+                        monitoring_hub_url,
+                        run_id,
+                        logging_level,
+                        sleep_dur,
+                        first_message=True)
+            except Exception:
+                pass
```

Why is it OK to silently discard exceptions here?